recipes/xr_fresh/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "xr_fresh" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mmann1123/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 000e241ca2a7ffa0f39f52aee53d4bf207cab04d8daa314a3842c661c2c51fd1
+
+build:
+  skip: true  # [py<37 or py>311]
+  skip: true  # [win]
```

Add the noarch part here.

```suggestion
  noarch: python
```